modules/host/storageobligations_smoke_test.go

```diff
@@ -199,18 +193,15 @@ func TestSingleSectorStorageObligationStack(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	err = ht.host.lockStorageObligation(so)
+	ht.host.lockStorageObligation(so.id())
 	if err != nil {
 		t.Fatal(err)
```

Comment:
checking the wrong `err` here, since `h.unlockStorageObligation(so.id())` doesn't return an error anymore